lib/rotp/arguments.rb

```diff
@@ -26,7 +27,15 @@ def options!
     end
 
     def default_options
-      OpenStruct.new time: true, counter: 0, mode: :time
+      o = Options.new
```

Comment:
Newer rubies make it possible to initialize it with keyword arguments, but since Ruby 2.3 is still supported I can't use that. That was only added in Ruby 2.5